packages/cfd/src/features/Containers/mt5-trade-modal.tsx

```diff
@@ -19,6 +19,7 @@ import { TTradingPlatformAccounts } from '../../Components/props.types';
 
 import { TCFDPasswordReset } from '../../Containers/props.types';
 import { CATEGORY, CFD_PLATFORMS, MARKET_TYPE, JURISDICTION } from '../../Helpers/cfd-config';
+import '../sass/cfd-dashboard.scss';
```

Comment:
If a specific styling is needed, it should be placed in a styles file inside the folder of the related components instead of importing them all from a main styles file.